src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -123,9 +108,7 @@ private static GameData fromMemento(final Memento memento) throws IOException {
     }
   }
 
-  private static GameData loadGameInSerializationFormat(
-      final InputStream inputStream,
-      final @Nullable String savegamePath)
+  private static GameData loadGameInSerializationFormat(final InputStream inputStream)
       throws IOException {
```

Comment:
With the removal of the `savegamePath` parameter, it looks like you'll have enough real estate to pull the exception declaration back onto the same line, as well.